### ISSUE TITLE
lateralisation of object counts

### ISSUE THREAD

**Issue by PolarBean** [Neural-Systems-at-UIO/nutil#272]
Several users and Trygve have requested the ability to have lateralised object counts + load. Can we calculate this in the output file for both left and right hemispheres.

**Comment by sharoncy** [comment]
This is currently supported with the application of hemisphere masks generated with the QuickMask software. This means that each dataset has to be processed twice, once for each hemisphere, to generate separate reports (object counts + loads) for each hemisphere. This is cumbersome, but is the solution that we have currently. The atlases themselves don't differentiate hemispheres so I got the impression this wasn't an easy thing to implement? But maybe there is a solution? Let's discuss sometime, it would be great if Nutil could do this without using masks.

**Comment by PolarBean** [comment]
We could perhaps create a custom atlas with different region codes for each hemisphere but it would be nice if it was an automatic feature for all atlases :)

**Comment by ingvildeb** [comment]
This could definitely be done by modifying the atlas itself but I agree it would be most elegant if it's implemented so that it's independent of the atlas delineations. I also think it's essential that the user can get both merged and lateralized counts within the same run.

**Comment by sharoncy** [comment]
I agree with Ingvild. As a minimum, there should be the option to generate two sets of reports - one corresponding to each side of the applied masks - within a single Nutil run. Nutil currently allows users to generate reports for either the left or right side in a single run. It should be a quick fix to implement support for generating both simultaneously.

**Comment by PolarBean** [comment]
Absolutely, and for efficiency would the sum of these reports be equal to the full run? that could be an idea

**Comment by Majpuc** [comment]
Lateralised version of Allen will be available. Need to be implemented in registration tools first.
